### PR TITLE
Issue 25

### LINKED ISSUE #25: MANE Transcripts
Suggest / promote use of MANE transcript and resultant protein sequence to top of validated sequences:

https://useast.ensembl.org/info/genome/genebuild/mane.html

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,23 @@ Variant Normalization relies on some local data caches which you will need to se
...
 pip install seqrepo
 mkdir -p data/seqrepo
-seqrepo -r data/seqrepo pull -i 2020-11-27
-chmod -R u+w data/seqrepo
+seqrepo -r data/seqrepo pull -i 2021-01-29
+sudo chmod -R u+w data/seqrepo
 cd data/seqrepo
 seqrepo_date_dir=$(ls -d */)
-mv $seqrepo_date_dir latest
+sudo mv $seqrepo_date_dir 2021-01-29
```
Comment (jsstevenson) [inline README.md:18]: remind me to make any necessary README changes to metakb issue-59 if we ever get around to reviewing it

**Review by jsstevenson — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (korikuzma) [reply README.md]: I changed it back to latest

**Review by jsstevenson — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ Variant Normalization relies on some local data caches which you will need to se
 From the _variant_ directory of the repository:
 ```
 pipenv sync
-curl ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/hgnc_complete_set.txt.txt > data/gene_symbols.txt
+curl ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/hgnc_complete_set.txt.txt > data/hgnc_gene_symbols.txt
+curl ftp://ftp.ncbi.nih.gov/refseq/H_sapiens/RefSeqGene/LRG_RefSeqGene > data/refseq_gene_symbols.txt
```
Comment (jsstevenson) [inline README.md:12]: if these aren't huge, could we just check + download them if they don't exist like we do w/ the normalizers?

**Review by jsstevenson — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (korikuzma) [reply README.md:12]: Oh you're right. I'll create a new issue.

**Pull request merged**